engines/procurement/spec/features/csv.feature

```diff
@@ -37,10 +38,43 @@ Feature: Exporting the data to a CSV-File
     Examples:
       | username  |
       | Barbara   |
-      | Roger     |
       | Hans Ueli |
 
   @csv
+  Scenario Outline: Export data
```

Comment:
@nadjaweisskopf we already have a scenario with the same name in this feature file. What is the purpose of this one?